standard/info/wikis/splatoon/info.lua

```diff
@@ -12,7 +12,7 @@ return {
 	name = 'Splatoon',
 	games = {
 		splatoon = {
```

Comment:
Keep the three id's consistent imo

Reply:
> Keep the three id's consistent imo

Is there a way to make aliases, since in the 2nd and 3rd game tournament pages input those two games in this way 
![image](https://user-images.githubusercontent.com/88981446/220117113-7f752d2b-cb98-49d6-8551-ebcd2dcc3c31.png) unless need to be bot edited on this